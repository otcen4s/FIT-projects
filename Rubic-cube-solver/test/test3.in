414
414
414
121 333 646 555
121 333 646 555
121 333 646 555
262
262
262