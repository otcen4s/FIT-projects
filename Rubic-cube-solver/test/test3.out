414
414
414
121 333 646 555
121 333 646 555
121 333 646 555
262
262
262

411
411
411
122 333 446 555
122 333 446 555
122 333 446 555
266
266
266

111
111
111
222 333 444 555
222 333 444 555
222 333 444 555
666
666
666

